package.json

```diff
@@ -7,10 +7,12 @@
     "test": "tests"
   },
   "engines": {
-    "node": "~4.2.0"
+    "node": "^4.2.0"
   },
   "scripts": {
+    "gulp": "gulp",
     "build": "gulp build",
+    "bundle": "gulp bundle",
```

Comment:
Can you please write some docs for this? The are others in `docs/en/05_Contributing/01_Code.md`